modules/querying-kubelet-status-on-a-node.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * support/verifying-node-health.adoc
+
+[id="querying-kubelet-status-on-a-node_{context}"]
+= Querying kubelet status on a node
+
+Review cluster node health status, resource consumption statistics and node logs. Additionally, query `kubelet` status on individual nodes.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. kubelet is managed using a systemd service on each node. Review kubelet status by querying the `kubelet` systemd service within a debug Pod:
+.. Start a debug Pod for a node:
++
+----
+$ oc debug node/my-node
+----
++
+.. Set /host as the root directory within the debug shell. The debug Pod mounts the node's root filesystem in `/host` within the Pod. By changing the root directory to `/host`, we can run binaries contained in the node's executable paths:
++
+----
+# chroot /host
+----
++
+[NOTE]
+====
+{product-title} {product-version} cluster nodes running Red Hat Enterprise Linux CoreOS are immutable and rely on Operators to apply cluster changes. Accessing cluster nodes using SSH is not recommended and nodes will be tainted as _accessed_. However, if the OpenShift API is not available, or `kubelet` is not properly functioning on the target node, `oc` operations will be impacted. In such situations, it is possible to access nodes using `ssh core@<node>.<cluster_name>.<sub_domain>.<domain>` instead.
```

Comment:
So, in all the places that you use {product-version}, I would just evaluate whether the version is truly necessary to say. Often times it's not, and just saying the product name is sufficient.

Reply:
Thank you. Resolved throughout the PR, where appropriate.